examples/viewer-integr-messaging.js

```diff
@@ -16,6 +16,17 @@
 
 
 /**
+ * @enum {string}
+ * @private
+ */
+const MessageType_ = {
```

Comment:
1. This has to be `var`. Unfortunately, this code has to be ES5 since it currently doesn't go via our compiler.
2. No need in `@private` or `_`.